src/main/resources/static/js/subscription.js

```diff
@@ -495,6 +492,31 @@ jQuery(document).ready(function () {
         table.responsive.recalc();
     });
 
+    function isEditAndDeleteButtonsDisabled(row, data) {
+        var disableEditDeleteButtons = true
+        var currentUser = sessionStorage.getItem("currentUser");
+
+        var isAnonymousSubscription = row.ldapUserName == null || row.ldapUserName == undefined || row.ldapUserName.length == 0
```

Comment:
create a function that computes if a subscription is anonymous or not. Format the logical chain too so that it is nicer formatted.

The same applies for isCurrentUserLoggedIn below

Reply:
Some improvements has been made to the code.